jsapp/js/components/submissions/submissionUtils.ts

```diff
@@ -108,16 +110,23 @@ export class DisplayResponse {
 
 /**
  * Returns a sorted object of transcript/translation keys
+ *
+ * Note: we omit returning `qual_note` questions.
  */
-export function sortAnalysisFormJsonKeys(additionalFields: Array<{source: string, dtpath: string}>) {
+function sortAnalysisFormJsonKeys(additionalFields: AnalysisFormJsonField[]) {
```

Comment:
Was this made before AnalysisFormJsonField was created/extended? There's a lot more mandatory fields than the parameter it replaced

Reply:
I was following the code that is using that function. It's actually being used only in one place, in `getSubmissionDisplayData`, which is passing `asset.analysis_form_json?.additional_fields` into it, and this prop is typed as `additional_fields: AnalysisFormJsonField[];`